test/pensions/config/childrenInformation.unit.spec.jsx

```diff
@@ -51,6 +51,29 @@ describe('Child information page', () => {
     expect(onSubmit.called).not.to.be.true;
   });
 
+  // Skipped because it's an array page with a ui:required
+  it.skip('should not require ssn if noSSN is checked', () => {
```

Comment:
This suffers from [this plight](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3690).